io.catenax.single_level_bom_as_built/2.0.0/SingleLevelBomAsBuilt

```diff
@@ -0,0 +1,152 @@
+#######################################################################
+# Copyright (c) 2022,2023 BASF SE
+# Copyright (c) 2022,2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+# Copyright (c) 2022,2023 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer ISST & Fraunhofer IML)
+# Copyright (c) 2022,2023 German Edge Cloud GmbH & Co. KG
+# Copyright (c) 2022,2023 Henkel AG & Co. KGaA
+# Copyright (c) 2022,2023 Mercedes Benz AG
+# Copyright (c) 2022,2023 Robert Bosch Manufacturing Solutions GmbH
+# Copyright (c) 2022,2023 SAP SE
+# Copyright (c) 2022,2023 Siemens AG
+# Copyright (c) 2022,2023 T-Systems International GmbH
+# Copyright (c) 2022,2023 ZF Friedrichshafen AG
+# Copyright (c) 2022,2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.single_level_bom_as_built:2.0.0#> .
+@prefix undefined: <urn:samm:io.catenax.single_level_bom_as_built:2.0.0#> .
```

Comment:
You can completely remove this line, as this is using word 'undefined'

Reply:
done

Reply:
@agg3fe Since we made this change the model is not validating anymore: 
![image](https://github.com/eclipse-tractusx/sldt-semantic-models/assets/116578665/eb675d75-6abb-4e21-8752-0e87adcaa793)
 any ideas why this might be the case? 
 @BenediktMuellerIML FYI

Reply:
@jonbckr 
Maybe the problem is, that we need to add another characteristic (e.g.: 'hasAlternativesCharacteristics') with the data type Boolean to the 'hasAlternatives' property so that the property does not point to the void?

I've tried it out on myself and when I add a characteristic with the Boolean data type, the problem no longer occurs:
<img width="360" alt="image" src="https://github.com/eclipse-tractusx/sldt-semantic-models/assets/139445668/73d7c737-a83c-47e7-a6f7-39c4f549f5ec">
<img width="532" alt="image" src="https://github.com/eclipse-tractusx/sldt-semantic-models/assets/139445668/0ab1bdd0-521e-4e70-aea7-e69278cafb38">

Reply:
Yes, makes sense to me. Thank you.